integration-tests/runTests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+TEST_NAMES='common'
```

Comment:
`runTests.sh` is only meant to run integration tests locally since on the CI every integration test is running in parallel. I'll put back the other tests.